packages/e2e.test.foundry-sdk-generator/tsconfig.json

```diff
@@ -7,5 +7,8 @@
   "include": [
     "./src/**/*"
   ],
+  "exclude": [
+    "./src/generatedNoCheck/**/*"
```

Comment:
This code that is generated just doesnt work in THIS project for type checking. It wasnt type checked before and isnt now to be consistent.